Corona-Warn-App/src/main/res/layout/activity_main.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
```

Comment:
What happens if we don't do this? Do we need to check all screens in the whole app now?

Reply:
Content will be displayed under status bar because of
`<item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>`